src/utils.ts

```diff
@@ -94,3 +95,7 @@ export function makeRandomHexString(length) {
 	}
 	return result;
 }
+
+export function isEmpty(str:string) {
+	return (!str || str === undefined || 0 === str.length);
```

Comment:
!str and str === undefined is basically testing the same thing. while str === undefined explicitely tests a var for undefined !str tests both null and undefined.